gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java

```diff
@@ -95,13 +104,14 @@ public VulkanEventTrack(QueryEngine qe, GpuInfo.VkApiEvent vkApiEvent) {
             data.commandBuffers[i] = row.getLong(5);
             data.submissionIds[i] = row.getLong(6);
             data.args[i] = args.getOrDefault(row.getLong(7), ArgSet.EMPTY);
+            data.dists[i] = row.getLong(8);
```

Comment:
Maybe a comment explaining what dists is.

Reply:
Thanks! Done in a new commit.